hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java

```diff
@@ -2893,7 +2909,7 @@ protected synchronized Properties getProps() {
         }
       }
     }
-    return properties;
+    return props;
```

Comment:
(I thought I posted a comment here, but somehow it got lost in github comments)
returning this object isn't necessary, and obviously the callers don't use the return value.